features/step_definitions/run_steps.rb

```diff
@@ -64,7 +64,12 @@
   expected_commands.map_column! 'COMMAND' do |command|
     command.gsub(sha_regex) do |sha_expression|
       commit_message = sha_expression.match(sha_regex).captures[0].strip
-      output_of "git reflog --grep-reflog='commit: #{commit_message.strip}' --format='%H'"
```

Comment:
If we get multiple results, how can we be sure that the first one is the correct one?